packages/terra-tag/tests/wdio/tag-spec.js

```diff
@@ -0,0 +1,130 @@
+/* global before, browser, Terra */
+const viewports = Terra.viewports('tiny', 'medium', 'large');
+
+describe('Tag', () => {
+  describe('Default', () => {
+    before(() => browser.setViewportSize(Terra.viewports('tiny')[0]));
+
+    describe('Default Tag', () => {
+      beforeEach(() => browser.url('/#/tests/tag-tests/default-tag'));
+
+      it('Checks if is-interactive class was not applied', () => {
+        expect(browser.getAttribute('span', 'className')).to.not.contains('is-interactive');
+      });
+
+      Terra.should.beAccessible();
+      Terra.should.matchScreenshot();
+      Terra.should.themeEachCustomProperty({
+        '--terra-tag-font-size': '25px',
+        '--terra-tag-color': 'red',
+        '--terra-tag-font-weight': '700',
+        '--terra-tag-padding-top': '1px',
+        '--terra-tag-padding-bottom': '1px',
+        '--terra-tag-padding-left': '1px',
+        '--terra-tag-padding-right': '1px',
+        '--terra-tag-border-style': 'dashed',
+        '--terra-tag-line-height': '20px',
+        '--terra-tag-margin-top': '1px',
+        '--terra-tag-margin-bottom': '1px',
+        '--terra-tag-margin-left': '1px',
+        '--terra-tag-margin-right': '1px',
+        '--terra-tag-text-decoration': 'underline',
+        '--terra-tag-background-color': 'green',
+        '--terra-tag-border-color': 'yellow',
+        '--terra-tag-box-shadow': '0 0 10px 10px blue',
+      });
+    });
+  });
+
+  describe('Default OnClick Tag', () => {
+    before(() => browser.url('/#/tests/tag-tests/default-on-click-tag'));
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+
+    it('Check if is-interactive class was applied', () => {
```

Comment:
Is this appropriate for a wdio tests? This feels more appropriate for a jest test.

Reply:
Even wdio tests allows us to check for the classes being applied. Other components have wdio tests similar to this one. But i still removed it as we have a similar jest test [here](https://github.com/cerner/terra-core/pull/1306/commits/b04c1661bc417ddbf31b7da23c28d95ec857e1df).